changelog.txt

```diff
@@ -18,6 +19,12 @@ Version 1.04.0
 - __FB_ASM__ wasn't #defined for x86_64 even though the -asm option can be used on x86_64 too
 - C backend: UDTs using Field=N were emitted incorrectly (causing gcc errors), if they contained a field of type of another UDT that didn't use Field=N.
 - False-positive "ambigious sizeof" warnings if the identifier could refer to a type or variable of that type (in this case the size is obviously the same, and it doesn't matter which one is used)
+- rtlib & gfxlib compilation was broken on NetBSD, Darwin and Solaris (header & include-path issues)
+- rtlib was ignoring const-correctness in hInit when calling tgetstr
+- Darwins linker was removing the __fb_errmsg symbol in rtlib
+- Correctly implemented fb_hGetExeName for FreeBSD, NetBSD, Darwin and Solaris (needed by gfxlib2)
+- Makefile wasn't able to detect the OS when building on Solaris
+- fbc was passing parameters to "as" that weren't valid on OS X
 
```

Comment:
I think some of these should be rewritten, maybe like this:

- rtlib & gfxlib compilation was broken on NetBSD, Darwin and Solaris (header & include-path issues)
- On Darwin, linking with rtlib failed due to __fb_errmsg being discarded
- Correctly implemented fb_hGetExeName for FreeBSD, NetBSD, Darwin and Solaris (needed by gfxlib2)
- The FB makefile can now detect the OS when building on Solaris
- fbc will now pass the proper parameters to "as" on OS X